test/commont-tools-test.cpp

```diff
@@ -0,0 +1,116 @@
+#include "gtest/gtest.h"
```

Comment:
Typo in the filename

Reply:
Renamed: commont-tools-test.cpp => tools-test.cpp